whitepaper.md

```diff
@@ -27,7 +27,9 @@ There are several elements mentioned in the definition:
 Our aim is to create a decentralized community platform governed by the community.
 First of all, community be ultimate source of decisions regarding how platform functions.
 
-In order to do this, community should be able to change also the rules how decisions are made in the community also.
+In order to do this, the community should be able to change also the rules how decisions are made in the community.
+
+> The sentence above seems out of place. Do you consider the AtomChange blockchain already such a change? If you want to ties this in to "forks are a way, but they are too divisive", I guess we can adapt this section.
```

Comment:
We can also discuss in comments to changes) 
Like this) You can find necessary changeset to comment on opening file -> Blame -> going to specific line and its changeset. 
Maybe you will find more convenient way. 

Regarding this specific question - yes, the point was that ultimately in an open network you cannot force community to do something it does not want to do, because if somebody will force, community will fork away. Forks are not totally bad, but in order to avoid unnecessary forks, community should have effective governance, and here we are with our project - trying to implement effective governance.